clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -161,7 +161,7 @@ scheduler_impl_tmpl_t<memref_t, reader_t>::get_reader(const std::string &path,
             // Skip the auxiliary files.
             if (fname == DRMEMTRACE_MODULE_LIST_FILENAME ||
                 fname == DRMEMTRACE_FUNCTION_LIST_FILENAME ||
-                fname == DRMEMTRACE_ENCODING_FILENAME || fname == DRMEMTRACE_V2P_FILENAME)
```

Comment:
I removed v2p.textproto from here, since its default location is now under aux/ instead of trace/, so the scheduler will not find it there.
Not sure if this change requires a minor version bump.
If it does, I'd rather put this back and wait for when we'll increase DR's version due to info.textproto containing more trace info as discussed offline.

Reply:
Maybe safer to leave it here just in case.  Probably even after we move the other files checked here we would keep the checks for back compat.

Reply:
Ok, re-added.